server/graph_service/dto/ingest.py

```diff
@@ -0,0 +1,25 @@
+from datetime import datetime
+
+from graphiti_core.nodes import EpisodeType  # type: ignore
+from pydantic import BaseModel, Field
+
+from graph_service.dto.common import Message
+
+
+class Episode(BaseModel):
```

Comment:
Why are we adding this new class rather than importing RawEpisode from graphiti?